lnwallet/wallet.go

```diff
@@ -59,7 +59,8 @@ var (
 	// transaction that would take the walletbalance below what we require
 	// to keep around to fee bump our open anchor channels.
 	ErrReservedValueInvalidated = errors.New("reserved wallet balance " +
-		"invalidated")
+		"invalidated: transaction would not leave sufficient funds to " +
```

Comment:
nit: how about `would leave insufficient reserve for fee bumping ...` instead of `not leave sufficient funds to fee bump ...`?